transforms/helpers/ast.ts

```diff
@@ -282,25 +269,38 @@ export function isEOPropertyForActionsObject(
 }
 
 interface EOActionsObjectExpression extends ObjectExpression {
-  properties: EOAction[];
+  properties: Array<EOActionMethod | EOActionProperty>;
 }
 
 function isEOActionsObjectExpression(
   u: unknown
 ): u is EOActionsObjectExpression {
-  return isNode(u, 'ObjectExpression') && u.properties.every(isEOAction);
+  return (
+    isNode(u, 'ObjectExpression') &&
+    u.properties.every(
+      (property) => isEOActionMethod(property) || isEOActionProperty(property)
+    )
+  );
 }
 
-export interface EOAction extends Property {
+export interface EOActionMethod extends ObjectMethod {
   key: Identifier;
-  value: FunctionExpression | Identifier;
 }
 
-function isEOAction(u: unknown): u is EOAction {
+export function isEOActionMethod(u: unknown): u is EOActionMethod {
+  return isNode(u, 'ObjectMethod') && isNode(u.key, 'Identifier');
+}
+
+export interface EOActionProperty extends ObjectProperty {
+  key: Identifier;
+  value: Identifier;
+}
+
+export function isEOActionProperty(u: unknown): u is EOActionProperty {
   return (
-    isNode(u, 'Property') &&
+    isNode(u, 'ObjectProperty') &&
```

Comment:
You definitely shouldn't do this (because scope, etc.) but I keep wishing that code like this was
more like:

```ts
return isNode(u, 'ObjectProperty', {
  key: 'Identifier',
  value: 'Identifier'
});
```

It's not **too** hard and might make a fun puzzle to do if you ever need a palate cleanser between
other tasks.

Reply:
That does sound fun. I wonder if checking them separately is easier to debug and step through though.